apps/emqx_exhook/include/emqx_exhook.hrl

```diff
@@ -43,4 +43,15 @@
 
 -define(DEFAULT_HOOK_PRIORITY, 0).
 
+-endif.
+
+-ifdef(TEST).
+
+-define(DEFAULT_CLUSTER_NAME, <<"emqxcl">>).
```

Comment:
Let's try to avoid using TEST macros in .hrl. This will make our code less clear.